### PR TITLE
Fixed issue with Qradar scripts

### PR DESCRIPTION
**Pull request by guykeller**
## Status 
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20289

## Description
The Qradar scripts were only compatible for one kind of CRE name default value.
This is now fixed.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

File: Scripts/QRadarGetCorrelationLogs/QRadarGetCorrelationLogs.py [hunk 1]
```diff
@@ -1,86 +1,106 @@
...
+                "identityip, CATEGORYNAME(category), PROTOCOLNAME(protocolid), " \
+                "LOGSOURCENAME(logsourceid){1} from events " \
+                "where \"CRE Name\" <> NULL AND INOFFENSE({2}) START '{3}'"
+    return query
+
+
+def main():
+    d_args = demisto.args()
+    is_cre_name_null = demisto.args().get("is_cre_name_null", "True")
+    QUERY = get_query(is_cre_name_null)
```
Comment (anara123) [inline Scripts/QRadarGetCorrelationLogs/QRadarGetCorrelationLogs.py]: there is no need to QUERY be with upper case

File: Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations_test.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import pytest
+from QRadarGetOffenseCorrelations import get_query
+
+
+@pytest.mark.parametrize('bool_val', ["False"])
+def test_get_query_cre_name_null_false(bool_val):
+    query_from_false = get_query(bool_val)
```
Comment (anara123) [inline Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations_test.py]: I think in tests you better mock the demisto.args() - pass is_cre_name_null=True and mock demisto.executeCommand and validate that the passed query contains `"\"CRE NAME\" <> NULL"` or you can compare the whole query

File: Scripts/QRadarGetCorrelationLogs/QRadarGetCorrelationLogs.py (hunk 1 above)
Comment (guykeller) [reply Scripts/QRadarGetCorrelationLogs/QRadarGetCorrelationLogs.py]: It was like that before and I'm not refactoring the script, just re wrote it to be more easily testable

File: Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations_test.py (hunk 2 above)
Comment (guykeller) [reply Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations_test.py]: How would that improve the test from your perspective?

Comment (anara123) [reply Scripts/QRadarGetOffenseCorrelations/QRadarGetOffenseCorrelations_test.py]: it will make it more end to end test.

**Pull request merged**